http/headers/feature-policy.json

```
@@ -1853,6 +1853,56 @@
             }
           }
         },
+        "web-share": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/web-share",
+            "support": {
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "81",
+                "notes": "Firefox 81 enables support for the Feature-Policy web-share, but does not support the WebShare API. See <a href='https://bugzil.la/1653481'>bug 1653481</a>."
```

If Firefox doesn't implement `navigator.share()`, then it seems to me that the header doesn't really have the specified effect of the policy, which is to define the conditions under which `navigator.share()` throws a `NotAllowedError`. This is _technically_ a case where `"partial_implementation"` should be `true`, but that's a bit misleading too. Taken together, I think it means we should be a bit more circumspect about "support" here. Perhaps we could say something like:

```suggestion
                "notes": "Firefox recognizes the <code>web-share</code> permissions policy, but this has no effect in versions of Firefox that do not support the <a href='https://developer.mozilla.org/docs/Web/API/Navigator/share'><code>share()</code></a> method."
```

I've dropped the bug link because, ideally, the `share()` method itself is going to be more interesting and actionable for web developers.